tests/microos/networkmanager.pm

```diff
@@ -61,8 +61,10 @@ sub run {
     # the below network confiration is used if on pvm
     %network_pvm = ();
     if (is_pvm) {
-        my $vm_mac = script_output qq(ip link show eth0 | grep -F 'link/ether' |awk '{print \$2}');
-        my $vm_ip = script_output qq(ip addr show eth0 | grep -F 'inet ' | awk '{print \$2}' |awk -F '/' '{print \$1}');
+        my $out = script_output("ls /sys/class/net");
```

Comment:
If you want to avoid the array it would be better to exclude the loopback. I assume that there is no case when a test job runs with 2 interfaces.

`ip -br link show | awk '! /lo/ {print $1}'` should be enough